ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -145,6 +148,11 @@ void WaylandWindow::ApplyPendingBounds() {
   connection_->ScheduleFlush();
 }
 
+void WaylandWindow::SetTouchFocus(bool focus) {
+  has_touch_focus_ = focus;
+  had_touch_focus_ = !has_touch_focus_;
```

Comment:
is there a spot we can reset this?

it will take a had_touch_focus = true, for 3-dot menu, and stick with true forever? (does it matter?)